### PR TITLE
Adds highlight for march 25

### PR DESCRIPTION
**Pull request by weatherhog**
### Please remember to

- [ ] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/changes/highlights/2022-03-25.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+date: 2022-03-25T10:00:00
+title: Highlights for the week ending March 25, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
```
Comment (marians) [inline src/content/changes/highlights/2022-03-25.md:10]:
```suggestion
## Apps

## User interfaces

The `kubectl gs template catalog` command provides a new flag `--visibility` to control whether the catalog should be visible in the web interface.

## Documentation
```

**Comment by gianfranco-l** [comment]
nothing from honeybadger for this week

File: src/content/changes/highlights/2022-03-25.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## User interfaces
+
+The `kubectl gs template catalog` command provides a new flag `--visibility` to control whether the catalog should be visible in the web interface.
+
+## Documentation
```
Comment (marians) [inline src/content/changes/highlights/2022-03-25.md:16]:
```suggestion
## Documentation

For customers using clusters on AWS or Azure, we added more information on how to spread workloads over several availability zones for better availability/resilience. If you haven't heard of `topologySpreadConstraints`, check our [example](https://docs.giantswarm.io/advanced/high-availability/multi-az/#example-pod-topology-spread-constraints-and-affinity).
```

**Comment by marians** [comment]
@JosephSalisbury I added another suggestions. That's it from us. Nothing from Rainbow this week.

I'm a bit puzzled by the lack of info on apps.

@stone-z What about https://docs.giantswarm.io/changes/managed-apps/starboard-app/v0.6.0/ ?

File: src/content/changes/highlights/2022-03-25.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+date: 2022-03-25T10:00:00
+title: Highlights for the week ending March 25, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
```
Comment (stone-z) [inline src/content/changes/highlights/2022-03-25.md:11]:
```suggestion

- [falco-app](https://github.com/giantswarm/falco-app) [v0.3.2](https://github.com/giantswarm/falco-app/blob/master/CHANGELOG.md#032---2022-03-25) changes the default driver from the kernel module to `ebpf`, supporting Linux kernel versions used in CAPI clusters.
- [security-pack](https://github.com/giantswarm/security-pack) [v0.0.1](https://github.com/giantswarm/security-pack/releases/tag/v0.0.1) has been released to the playground catalog as convenient way to install multiple [security pack](https://docs.giantswarm.io/app-platform/apps/security/) components at once.
- [starboard-app](https://github.com/giantswarm/starboard-app) [v0.6.0](https://github.com/giantswarm/starboard-app/blob/master/CHANGELOG.md#060---2022-03-23) updates to Starboard version 0.14.1 and uses the newer Trivy 0.24.0 backend.
- [trivy-app](https://github.com/giantswarm/trivy-app) [v0.2.0](https://github.com/giantswarm/trivy-app/blob/master/CHANGELOG.md#020---2022-03-23) updates to Trivy version 0.24.0.

```

Comment (marians) [reply src/content/changes/highlights/2022-03-25.md:11]: There we go :)

**Pull request merged**